src/assets/CollectiblesController.ts

```diff
@@ -47,11 +47,14 @@ import { compareCollectiblesMetadata } from './assetsUtil';
  * @property externalLink - External link containing additional information
  * @property creator - The collectible owner information object
  * @property isCurrentlyOwned - Boolean indicating whether the address/chainId combination where it's currently stored currently owns this collectible
+ * @property isTransacting - Boolean indicating if the Collectible is being transacting
  */
 export interface Collectible extends CollectibleMetadata {
   tokenId: string;
   address: string;
   isCurrentlyOwned?: boolean;
+  isTransacting?: boolean;
+  transactionId?: string;
```

Comment:
Is it possible for `isTransacting` to be true and `transactionId` to be undefined? Conversely, is it possible for `isTransacting` to be false and `transactionId` to be set? I'm wondering if `transactionId` can already be used to determine whether the collectible is being transacted or not.

Reply:
Yes! It will be very better that way! Thank you a lot!